client.go

```diff
@@ -30,6 +30,8 @@ import (
 	"time"
 )
 
+var MARATHON_TOKEN = ""
```

Comment:
We shouldn't work with globals here -- they pose a risk for concurrent usage. A better place is likely our [configuration](https://github.com/gambol99/go-marathon/blob/697a616b3996092969b86950b1e555e3b4391945/config.go#L29).